host-bin/enter-chroot

```diff
@@ -436,9 +436,21 @@ elif [ ! -f "$shares" ]; then
 downloads ~/Downloads
 EOF
 fi
+
 # Bind-mount the stuff in $CHROOT/etc/crouton/shares, unless NOLOGIN is set
 if [ -z "$NOLOGIN" -a -f "$shares" ]; then
-    localdownloads='/home/chronos/user/Downloads'
+    activeuser=''
+    if [ -x "/usr/bin/jq" ]; then
+        activeuser="$(jq -r .LastActiveUser '/home/chronos/Local State')"
+    fi
+    if [ -n "$activeuser" ]; then
+        localdownloads="$(jq -r '.profile.info_cache | to_entries[]
+          | select(.value.user_name == "'$activeuser'")
+          | "/home/chronos/\(.key)/Downloads"' '/home/chronos/Local State')"
```

Comment:
I'm not convinced this is better (also not certain it is the correct way of doing it), but if we want to get rid of the jq dependency:
```
localdownloads="$(grep -l '"account_id":\s*"'"$activeuser"'"' /home/chronos/u-*/Preferences | sed 's/Preferences$/Downloads/;q')"
```